src/components/sticky-container.js

```diff
@@ -26,6 +25,7 @@ var StickyButton = React.createClass({
     }
   },
   componentDidMount: function() {
+    window.scrollTo(0, 0);
```

Comment:
Ensure this line doesn't get server compiled. did mount and initial state are the same for what we need, but the server never mounts.